dds_web/api/project.py

```diff
@@ -124,10 +124,15 @@ def post(self):
         try:
             project.project_statuses.append(new_status_row)
             db.session.commit()
-        except (sqlalchemy.exc.SQLAlchemyError) as err:
+        except (sqlalchemy.exc.OperationalError, sqlalchemy.exc.SQLAlchemyError) as err:
             flask.current_app.logger.exception(err)
             db.session.rollback()
-            raise DatabaseError(message="Server Error: Status was not updated") from err
+            raise DatabaseError(
+                message=str(err),
+                alt_message="Database seems to be down."
+                if isinstance(err, sqlalchemy.exc.OperationalError)
+                else "Server Error: Status was not updated",
+            ) from err
```

Comment:
```suggestion
            raise DatabaseError(
                message=str(err),
                alt_message=(
                    "Status not updated" + 
                    (
                        "Database malfunction." 
                        if isinstance(err, sqlalchemy.exc.OperationalError)
                        else "Server Error: Status was not updated"
                     )
                )
             ) from err
```
You probably need to run the linting though, this will likely make it complain...